### DISCUSSION TITLE
OpenGL custom zoom limits

### DISCUSSION THREAD

**Discussion by JeffreyvanNijnatten** [OpenRCT2/OpenRCT2#15793] [Feature requests]
Using OpenGL instead of of the Software (via hardware) renderer allows us to be able to zoom in more than originally possible. However, especially with a scalefactor set, the maximum zoom levels can be a little too much for the user experience.

**My suggestion is to allow the user to set a custom default zoom limit in the settings.**

Additionally, if desired. Maybe the extra zoom limits could be accessed holding CTRL + Scrolling for example?

**Comment by J-PIE-314** [comment]
#22867 and #23152 are similar.
